### ISSUE TITLE
[Accessibility]A11y_VSCode_CommandPalette_ColorContrast: Color Contrast ratio is less than 4.5:1 for the placeholder in "Command Palette"

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52254]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
ColorContrast,MAS1.4.3

**Repro Steps:**
1.Navigate to Settings icon in Activity Bar->Press Enter->Select "Command Palette" from dropdown.
2.Open color contrast analyzer
3.Observed that,color contrast ratio is less than 4.5:1 ratio.

**Actual:**
Checking through color contrast analyzer, the color contrast ratio for the placeholder "Type ? to get help on the actions you can take from here" is 2.3:1

**Expected:**
The color contrast ratio should be equal to or greater that 4.5:1, here the placeholder should be equal or greater that 4.5:1 ratio

**Recommendations:**
Highlight the background color and make the foreground text in light color or vice-versa.
or,
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to clearly see the text written inside the controls.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)

**Attachment For Reference:**
![accessibility a11y_vscode_commandpalette_colorcontrast](https://user-images.githubusercontent.com/39204909/41577568-3f23ecf2-73ab-11e8-85f1-5032bb5cd870.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
Duplicate #51831
